core/src/test/java/de/danoeh/antennapod/core/sync/GuidValidatorTest.java

```diff
@@ -0,0 +1,16 @@
+package de.danoeh.antennapod.core.sync;
+
+import junit.framework.TestCase;
+
+public class GuidValidatorTest extends TestCase {
+
+    public void testIsValidGuid() {
+        assertTrue(GuidValidator.isValidGuid("skfjsdvgsd"));
+    }
+
+    public void testIsInvalidGuid() {
+        assertFalse(GuidValidator.isValidGuid(""));
+        assertFalse(GuidValidator.isValidGuid(" "));
```

Comment:
I think another common case would be that podcasters do something like this:

```
    <guid>
    </guid>
```

So there could be both newlines and spaces. Does `trim()` remove them both?

Reply:
extended the test with your examples